bl-plugins/rss/plugin.php

```diff
@@ -32,6 +32,11 @@ public function form()
 
 		return $html;
 	}
+	
+        private function urlRSS($url)
```

Comment:
maybe this function call it something like `encodeURL`

Reply:
If you meant PHP urlencode() it encodes slashes and punctuations as well. For RSS we need only encode non-ASCII symbols.

Reply:
I mean the name of the function, instead of `urlRSS` change it for `encodeURL`.

Reply:
No problem. Done.